core/domain/question_domain.py

```diff
@@ -229,3 +237,64 @@ def update_question_data(self, question_data):
             question_data: dict. A dict representing the question data.
         """
         self.question_data = question_data
+
+    def add_skill(self, skill_name, user_id):
+        """Adds the question id to the question list of the appropriate skill.
+
+        Args:
+            skill_name: str. The name of the skill.
+            user_id: str. The id of the user.
+        """
+        collection_services.update_collection(
+            user_id, self.collection_id, [{
+                'cmd': collection_domain.CMD_ADD_QUESTION_ID_TO_SKILL,
+                'skill_name': skill_name,
+                'question_id': self.question_id
+            }], 'Add a question_id to skill')
```

Comment:
I think it's not a bad idea to make this more explicit, e.g. "Add a new question with ID {{question_id}} to skill {{skill_name}} ({{skill_id}})." Ditto for remove_skill.

Reply:
Updated to "Add a new question with ID {{question_id}} to skill {{skill_id}}." Reduces the call for skill_name.